src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  --color: #00acdc;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
The 'font-family' is already defined in the 'html' selector. There's no need to redefine it here unless you want to specify a different font for the 'body' element.